adoc/admin-updates.adoc

```diff
@@ -241,6 +241,23 @@ kubectl -n kube-system annotate ds kured weave.works/kured-node-lock='{"nodeID":
 
 This command modifies an annotation (`annotate`) on the daemonset (`ds`) named `kured`.
 
+After you have done this, you will have to manage reboots yourself. In order to do this, you will have to follow some steps whenever you want to issue a reboot for a node. First of all, you will have to `cordon` and `drain` the node:
+
+----
+kubectl cordon NODE
+kubectl drain --force=true --delete-local-data=false --ignore-daemonsets=false NODE
```

Comment:
I would also add the following options `--grace-period=600 --timeout=900s` (values I use for CAP) with a note explaining that this has to configured in regards of the application running. App with data may require more time to stop successfully.

Without this, draining could hang if a pod can't be terminated.

Reply:
So it will grant the app 600s to shut down (no time unit given here?) and 900s (seconds) until it forces the removal, right?

Reply:
I would also add a note about `--delete-local-data=false`: Continue even if there are pods using emptyDir (local data that will be deleted when
the node is drained).

For example, this is required when a node is running a `metrics-server` pod.

Reply:
> So it will grant the app 600s to shut down (no time unit given here?) 
yes

> and 900s (seconds) until it forces the removal, right?

No, it will just fail